perllib/FixMyStreet/App/Controller/Alert.pm

```diff
@@ -196,13 +196,18 @@ sub create_alert : Private {
         $alert->insert();
     }
 
+    $c->stash->{alert} = $alert;
+
     if ( $c->user_exists && ($c->user->id == $alert->user->id || $c->stash->{can_create_for_another})) {
         $alert->confirm();
+        if ($c->stash->{can_create_for_another}) {
```

Comment:
This will send the email to a staff user with this permission who is creating an alert for themselves as well. I guess it should check the alert user is not the same as the logged in user.